### PR TITLE
add lodestar Mainnet to stakers UI

### PR DESCRIPTION
**Pull request by alexpeterson91**
Add Lodestar Mainnet to stakersUI

Not to be merged until lodestar mainnet is published, web3signer is updated, and testing is complete

### CODE REVIEW

**Comment by alexpeterson91** [comment]
> this should be unblocked as per https://github.com/dappnode/DNP_DAPPMANAGER/issues/1422 tracker

It's been marked as blocked solely so it didn't get merged after approval before it's ready. It's not anything actually blocking merging. And I'm not just gonna go around to each PR and deselect the label and rename the title to remove the blocked notification. It's not necessary and a waste of time. If it's ready to merge (then the Core team should give it an approving review so I can safely merge it. Or just merge it themselves. But it hasn't been reviewed yet so that's what's actually blocking it I already jumped the gun merging lodestar prater with approval so there's no way I'm gonna do that again especially for something so critical as this. 

If there's something actually blocking the merging process it's got nothing to do with the title including the word blocked or the label `blocked`, rather the branch protections which require an approving review from the @dappnode/core team and passing CI checks. It can be overridden by admins such as myself but shouldn't be especially for new ETH clients until specifically @pablomendezroyo determines it's time to add the package to the dappmanager. 

The reason I added the blocked note was because when I added lodestar prater with approving reviews etc it had to be reverted because Pablo needed to publish another release prior to one with Lodestar prater and the whole core team wanted more testing done. 

So everything else not yet ready to be merged and published I marked as blocked to avoid that happening again. He and he alone decides when each release is ready and what will make it into each release. 

And all lodestar prepping/prepped PRs I made predated the tracking issue above and were tracked on a much older project; with dozens of other related PRs and issues tracked in this project I made 7 months ago for adding the lodestar consensus clients. They were listed as done awaiting for final approval there. And just here in its current state.

**Comment by nflaig** [comment]
It not about merging this, just in regard to the blockers mentioned in PR description.

> Not to be merged until lodestar mainnet is published, web3signer is updated, and testing is complete

Just added this comment so @pablomendezroyo or anyone else looking at the PR does not have to wonder if conditions mentioned in the PR description are now fulfilled or not.

I was just going through [checklist](https://hackmd.io/YYZdWikXT72exnDOil7OUQ) to verify everything and test on my stack to make sure Lodestar mainnet works. After that I just dropped the comment here to keep things up-to-date. Sorry if that caused confusing 🙏 

> title including the word blocked or the label blocked

that's definitely great approach, allows to cross reference the PR and preventing it from getting merged 👍 

>  I made predated the tracking issue above and were tracked on a much older project

the tracking issue is amazing, big shoutout to you and @dsimog01 for all the work you have done to get Lodestar on DAppNode

**Review by pablomendezroyo — APPROVED** [review]

**Review by dsimog01 — APPROVED** [review]

**Pull request merged**
